README.md

```diff
@@ -1,2 +1,44 @@
-# CapstoneProject1
-Capstone Project 1 Module 1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+Capstone 1
+
+# Online Version
+
+https://gealsanchez.github.io/Capstone1/
+
+## Built With
+
+- HTML & CSS & Javascript
+- GitHub
+
+## Live Pull Request
+
+[Live Pull Request Link](https://github.com/gealsanchez/portfolio/pull/1)
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/gealsanchez)
+- Twitter: [@twitterhandle](https://twitter.com/gealsanchez)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/gerson-sanchez-88309b57/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Wali Muhammad
```

Comment:
- [ ] Kindly add credits to the designer (Cindy Shin) as mentioned in the requirements 👍